cytotable/utils.py

```
@@ -923,6 +924,26 @@ def find_anndata_metadata_field_names(
     return numeric_fields, non_numeric_fields
 
 
+def _glob_follow_symlinks(start: Path, pattern: str) -> Iterator[Path]:
+    """
+    Like ``Path.glob(pattern)``, but follows symlinked directories on every
+    Python version CytoTable supports. ``Path.glob`` only gained
+    ``recurse_symlinks=True`` in 3.13; on 3.10–3.12 we walk the tree with
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)